pyproject.toml

```diff
@@ -97,6 +97,13 @@ todoist = "bugwarrior.services.todoist:TodoistService"
 [project.entry-points."ini2toml.processing"]
 bugwarrior = "bugwarrior.config.ini2toml_plugin:activate"
 
+[tool.ruff.format]
+quote-style = "single"
```

Comment:
ruff default value, and the usual python standard (statement open to debate) is to use double quotes, though there is nothing wrong with using single quotes I guess

Reply:
By switching to "preserve" the diff is reduced to >2k lines. I'm not sure there's much of an advantage to enforcing project-wide quote consistency but I'm open to revisiting the question.